gce-production-1/main.tf

```diff
@@ -170,9 +174,23 @@ resource "google_project_iam_member" "staging_1_workers" {
   member  = "serviceAccount:${element(data.terraform_remote_state.staging_1.workers_service_account_emails, count.index)}"
 }
 
+resource "google_project_iam_member" "staging_1_warmer" {
+  count   = "${length(data.terraform_remote_state.staging_1.warmer_service_account_emails)}"
+  project = "${var.project}"
+  role    = "roles/compute.imageUser"
+  member  = "serviceAccount:${element(data.terraform_remote_state.staging_1.warmer_service_account_emails, count.index)}"
+}
+
 resource "google_project_iam_member" "production_2_workers" {
   count   = "${length(data.terraform_remote_state.production_2.workers_service_account_emails)}"
   project = "${var.project}"
   role    = "roles/compute.imageUser"
   member  = "serviceAccount:${element(data.terraform_remote_state.production_2.workers_service_account_emails, count.index)}"
 }
+
+resource "google_project_iam_member" "production_2_warmer" {
```

Comment:
only other nit: why we name this member (and production workers) with `2` while staging  has`1`, given that we are in `gce-production-1`s main.tf?  Any significance?

Reply:
This resource only exists in `gce-production-1` and the purpose it to grant cloud images usage access to the _other_ `gce-.+` projects, which is why this resource has this name.